### ISSUE TITLE
Provide binary for ARM

### ISSUE THREAD

**Issue by kamilogorek** [getsentry/sentry-cli#676]
ref: https://github.com/getsentry/sentry-cli/issues/592

**Comment by Alexandero89** [comment]
Would really appreciate a binary for ARM.

Raspberry 4 with 4 Gig RAM and SSD should work fine with Sentry.

**Comment by fastlorenzo** [comment]
Same here, preferably an ARM binary for musl as well (alpine docker)

**Comment by omaraltayyan** [comment]
please note this has increased in importance with the release of AWS's new Gravaton2 processors into EC2 mainstream instances (T4G), so it's becoming a necessity to support ARM in popular packages.

**Comment by albertshaw** [comment]
Any luck? sentry-webpack-plugin does not work with Macbook M1

**Comment by omaraltayyan** [comment]
> Any luck? sentry-webpack-plugin does not work with Macbook M1

You can build sentry cli on arm from source, there's still no provided binary for it.

**Comment by gersakbogdan** [comment]
Until this will be available you can use my repo: https://github.com/gersakbogdan/sentry-cli-Linux-arm64
Those are built from source.

**Comment by zackify** [comment]
would be great to have this so that I can use sentry inside docker on M1

**Comment by mercs600** [comment]
the same issue here - with installation inside of docker

**Comment by Mati365** [comment]
bump

**Comment by kamilogorek** [comment]
Released as `1.63.0`. Cheers!

**Comment by tmakin** [comment]
@kamilogorek I've just tried the new 1.63 release on an EC2 ARM instance but the binary still appears to be missing. Here is my output from `yarn --verbose`
```
Error: Unable to download sentry-cli binary from https://downloads.sentry-cdn.com/sentry-cli/1.63.0/sentry-cli-Linux-arm64.
Server returned 403: Forbidden.
```

It's looking for `arm64` which seems to conflict with the `aarch64` naming convention [here](https://github.com/getsentry/sentry-cli/blob/master/.craft.yml#L41).

**Comment by kamilogorek** [comment]
Released as `1.63.1`
